mine.js

```diff
@@ -10,17 +10,29 @@ const tmp = require('tmp')
 const path = require('path')
 const fs = require('fs')
 const spawn = require('child_process').spawn
+const IPFS = require('./lib/ipfs')
 
-async function checkForModels (mineAddress, contractAddress, web3, ipfs) {
+async function checkForModels (mineAddress, contractAddress, web3) {
+  IPFS.connect(function(err, ipfs) {
+    if (err) {
+      console.error(`ipfs error: `, err)
+      return
+    }
+
+    trainModels(mineAddress, contractAddress, web3, ipfs)
+  })
+}
+
+async function trainModels (mineAddress, contractAddress, web3, ipfs) {
   const sonar = new Sonar(web3, contractAddress, mineAddress)
 
   console.log(`🔎️  Looking for models to train at ${contractAddress} for mine ${mineAddress}`)
   const modelCount = await sonar.getNumModels()
-  console.log(`💃  ${modelCount} models found`)
+  console.log(` 💃  ${modelCount} models found`)
```

Comment:
no unrelated changes in PRs. Do you have issues with the emoji formatting on your CLI? It works for me